getting-started/quickstart-guides/php.md

```diff
@@ -42,425 +42,1483 @@ It will prompt a few options. Select defaults by pressing enter.
 require_once __DIR__ . '/vendor/autoload.php';
 
-use ImageKit\ImageKit;
-
+use ImageKit\ImageKit;  
+  
 // For demonstration purposes, the documentation would use https://ik.imagekit.io/demo as urlEndpoint
 $imageKit = new ImageKit(
-    "{publicKey}",
-    "{publicKey}",
-    "{urlEndpoint}" 
+    "publicKey",
+    "privateKey",
+    "urlEndpoint"
 );
-
 ```
+* `publicKey` and `privateKey` parameters are required as these would be used for all ImageKit API, server-side upload, and generating token for client-side file upload. You can get these parameters from the developer section in your ImageKit dashboard - [https://imagekit.io/dashboard#developers](https://imagekit.io/dashboard#developers).
+* `urlEndpoint` is also a required parameter. You can get the value of URL-endpoint from your ImageKit dashboard - [https://imagekit.io/dashboard#url-endpoints](https://imagekit.io/dashboard#url-endpoints).&#x20;
 
-* `urlEndpoint` is the required parameter. You can get the value of URL-endpoint from your ImageKit dashboard - [https://imagekit.io/dashboard#url-endpoints](https://imagekit.io/dashboard#url-endpoints).&#x20;
-* `publicKey` and `privateKey` parameters are also required as these would be used for all ImageKit API, server-side upload, and generating token for client-side file upload. You can get these parameters from the developer section in your ImageKit dashboard - [https://imagekit.io/dashboard#developers](https://imagekit.io/dashboard#developers).
-
-## Generating url for rendering images
-
-#### URL for image with relative path&#x20;
-
+#### URL Generation
 ```php
-$imageURL = $imageKit->url(['path' => '/default-image.jpg']);
-
-echo('Url : ' . $imageURL); 
-// https://ik.imagekit.io/demo/default-image.jpg
+// For URL Generation
+$imageURL = $imageKit->url(
+    [
+        'path' => '/default-image.jpg',
+    ]
+);
+echo $imageURL;
 ```
 
-![](../../.gitbook/assets/default-image.jpeg)
-
-#### URL for image with relative path and custom URL-endpoint
-
+#### File Upload
 ```php
-$imageURL = $imageKit->url([
-      'urlEndpoint' => 'https://ik.imagekit.io/test',
-      'path' => '/default-image.jpg',
+// For File Upload
+$uploadFile = $imageKit->upload([
+    'file' => 'file-url',
+    'fileName' => 'new-file'
 ]);
+```
 
-echo('Url : ' . $imageURL); 
-// https://ik.imagekit.io/test/default-image.jpg
+#### Response Structure
+```json
+{
+    "error": {},
+    "result": {},
+    "responseMetadata": {
+        "headers": {},
+        "raw": {},
+        "statusCode": {statusCode}
+    }
+}
 ```
 
-#### &#x20;URL for image with absolute url&#x20;
+## Generating url for rendering images
+ImageKit provides inbuild media storage and integration with external origins. Refer to the [Documentation](https://docs.imagekit.io/integration/url-endpoints) to learn more about URL endpoints and external [Image Origins](https://docs.imagekit.io/integration/configure-origin) supported by ImageKit.  
```

Comment:
Typo. The correct word is `inbuilt`. Go through the text once, but typos will be taken care of in the final review before we merge it to master.